src/main/resources/application.yml

```diff
@@ -60,6 +60,10 @@
       # How often to check?
       interval-seconds: 60
 
+vscode:
+  extension:
+    version: unknown
+  version: unknown
```

Comment:
These configs are used by the `SidecarInfo`. We define defaults here, so that we can use them in the `client.id` properties of the Admin, Consumer, and Producer client.